fish-rust/src/expand.rs

```diff
@@ -0,0 +1,39 @@
+use crate::wchar::{EXPAND_RESERVED_BASE, EXPAND_RESERVED_END};
+
+/// Private use area characters used in expansions
+#[repr(u32)]
+pub enum ExpandChars {
+    /// Character representing a home directory.
+    HomeDirectory = EXPAND_RESERVED_BASE as u32,
+    /// Character representing process expansion for %self.
+    ProcessExpandSelf,
```

Comment:
Gonna use `pub const char` instead of the enum, I think that matches the use better

Reply:
just saw there is a From implementation for `char` which helps.. still, these are better of as char constants, the enum doesn't give us anything (since it's not exhaustive for the switches we use)